AnnoDesigner/AnnoCanvas.xaml.cs

```diff
@@ -1522,21 +1576,30 @@ protected override void OnKeyDown(KeyEventArgs e)
         /// <summary>
         /// Checks whether the user is pressing the control key.
         /// </summary>
-        /// <returns></returns>
+        /// <returns><c>true</c> if the control key is pressed, otherwise <c>false</c>.</returns>
         private static bool IsControlPressed()
         {
-            return Keyboard.Modifiers.HasFlag(ModifierKeys.Control) || Keyboard.Modifiers.HasFlag(ModifierKeys.Shift);
+            return Keyboard.Modifiers.HasFlag(ModifierKeys.Control);
```

Comment:
We can use `Keyboard.Modifiers & ModifierKeys.Control != 0` here instead. `HasFlag` is about 10-20x slower.

I realise some of the comments previously said that this probably wasn't an issue, but it has since occured to me that although in this case (for this PR) we are using the method from an event, that might not be the case in future.